actionlib_msgs/GoalID id

# List of keypoints that need to be in the path
uint64[] path

# Duration to wait before/after goal start/completion
duration pauseBefore
duration pauseAfter
---
uint8 SUCCESS = 0
uint8 GOAL_IN_PORGRESS = 1

uint8 returnCode